lang/po/new/cs.po

```diff
@@ -593,7 +590,7 @@ msgstr "Vyber filament:"
 # MSG_FILAMENT c=17
 #: messages.c:35
 msgid "Filament"
-msgstr ""
+msgstr "Filament"

```

Comment:
In case that the translation is the same as the source text, please leave it empty to save some space.